### PR TITLE
Add third builder - relay validator

### PR DESCRIPTION
**Pull request by piwonskp**
As being the relay validator affects performance of the builder the PR adds another builder to act as the relay validator

### CODE REVIEW

**Comment by kamilchodola** [comment]
@piwonskp 
Change all EL and CL version from static one to "latest" to decrease maintenance.

File: scripts/kurtosis/network_params.json [hunk 1]
```diff
@@ -56,29 +56,66 @@
...
+                "--disable-peer-scoring"
+            ],
+            "el_extra_params": [
+                "--builder",
+                "--builder.remote_relay_endpoint=http://mev-relay-api:9062",
+                "--builder.beacon_endpoints=http://cl-3-lighthouse-geth:4000",
+                "--builder.bellatrix_fork_version=0x30000038",
+                "--builder.genesis_fork_version=0x10000038",
+                "--builder.genesis_validators_root=GENESIS_VALIDATORS_ROOT_PLACEHOLDER",
+                "--miner.extradata=The_Dev_version_of_builder",
```
Comment (kamilchodola) [inline scripts/kurtosis/network_params.json]: This shoudl be Baseline

File: scripts/kurtosis/network_params.json [hunk 2]
```diff
@@ -72,13 +109,12 @@
         "mev_boost_image": "flashbots/mev-boost:1.6",
         "mev_builder_image": "flashbots/builder:latest",
```
Comment (kamilchodola) [inline scripts/kurtosis/network_params.json:107]: @piwonskp Shall we specify somehow Lighthouse version for this builder as well?

Comment (piwonskp) [reply scripts/kurtosis/network_params.json:107]: I don't think it matters. It's latest by default: https://github.com/kurtosis-tech/ethereum-package/blob/main/src/package_io/input_parser.star#L443. I don't think it'll ever change in ethereum-package. If it does I guess it'll be for a good reason

Comment (piwonskp) [reply scripts/kurtosis/network_params.json:107]: We could even ommit it in dev & baseline builder if we use latest imo. If ethereum-package changes provider it'll be automatically updated then

Comment (kamilchodola) [reply scripts/kurtosis/network_params.json:107]: Yeah so we can simplify if latest is default :)

Comment (piwonskp) [reply scripts/kurtosis/network_params.json:107]: Done

**Review by kamilchodola — APPROVED** [review]

**Pull request merged**